src/loader/cmti.ml

```diff
@@ -160,12 +160,16 @@ let read_label_declaration env parent ld =
   let type_ = read_core_type env ld.ld_type in
     {id; doc; mutable_; type_}
 
-let read_constructor_declaration_arguments env parent arg =
+let read_constructor_declaration_arguments env _parent arg =
```

Comment:
Same thing with `_parent`, suggest `ignore parent;` in 4.02 code.

Reply:
Fixed 🆗